### PR TITLE
fixed some typos

### PR DESCRIPTION
**Pull request by nouranali**
I have fixed some typos and grammar faults

### CODE REVIEW

File: doc/readme/DEVELOPING.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Developing Shogun
 
-This is a very basic list of things how to get started hacking Shogun.
-Your first steps should be to
+This is a very basic checklist on how to get started in hacking Shogun,
```
Comment (gf712) [inline doc/readme/DEVELOPING.md]: `how to get started with...`

File: doc/readme/DEVELOPING.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Developing Shogun
 
-This is a very basic list of things how to get started hacking Shogun.
-Your first steps should be to
+This is a very basic checklist on how to get started with Shogun,
+You should:
```
Comment (gf712) [inline doc/readme/DEVELOPING.md]: Maybe just drop the `you should:`, i.e. `This is a very basic checklist on how to get started with Shogun:`

Comment (nouranali) [reply doc/readme/DEVELOPING.md]: great, will do

File: doc/readme/DEVELOPING.md [hunk 3]
```diff
@@ -1,13 +1,12 @@
 # Developing Shogun
 
-This is a very basic list of things how to get started hacking Shogun.
-Your first steps should be to
+This is a very basic checklist on how to get started with Shogun :
```
Comment (gf712) [inline doc/readme/DEVELOPING.md]:
```suggestion
This is a very basic checklist on how to get started with Shogun:
```

File: doc/readme/DEVELOPING.md (hunk 1 above)
Comment (nouranali) [reply doc/readme/DEVELOPING.md]: done

**Pull request merged**